exocom-server/src/message-translator/message-translator.ls

```diff
@@ -0,0 +1,29 @@
+class MessageTranslator
+
+  # Returns the message name to which the given service would have to subscribe
+  # if it wanted to receive the given message expressed in its internal form.
+  #
+  # Example:
+  # - service "tweets" has internal namespace "text-snippets"
+  # - it only knows the "text-snippets.create" message
+  # - the external message name that it has to subscribe to is "tweets.create"
+  public-message-name: ({internal-message-name, client-name, internal-namespace}) ->
+    message-parts = internal-message-name.split '.'
```

Comment:
You could inline this variable to improve readability (allows you to get rid of the `switch` statement and do pure pattern matching in this method